packages/frontend-v2/pages/index.tsx

```diff
@@ -1,23 +1,13 @@
 import type { NextPage } from "next";
-import { SeasonEnum } from "@graduate/common";
+import { Flex, Heading } from "@chakra-ui/react";
 
 const Home: NextPage = () => {
-  // using a type from our common package
-  const seasons: SeasonEnum[] = [
-    SeasonEnum.FL,
-    SeasonEnum.SP,
-    SeasonEnum.S1,
-    SeasonEnum.S2,
-  ];
-
   return (
-    <>
-      <h1>GraduateNU Landing Page:</h1>
-      <h2>Seasons typed using types from our common package!</h2>
-      {seasons.map((season) => (
-        <p key={season}>{season}</p>
-      ))}
-    </>
+    <Flex width="full" justifyContent="center" my="xl">
+      <Heading color="primary.main" size="xl">
```

Comment:
size="xl" makes the Heading automatically responsive(size decreases on smaller screen sizes, and vice-versa) in a way that is consistent with the entire application(all components follow the same responsive guidelines).